### DISCUSSION TITLE
merging Seurat objects with slightly different genomes

### DISCUSSION THREAD

**Discussion by shinlin77** [stuart-lab/signac#851] [Q&A]
I have a lot of Multiome files to merge in a big analysis.  All but one comes from lab A.  One last one comes from lab B.  I can successfully merge all the files from lab A.  When I then try to merge the one from lab B, merge hangs up (it will not finish after several hours).  It gets past the first bar but then nothing happens.  When I look at CPU usage, it is minimal at that point.

> total = merge(total, sample)
  |======================================================================| 100%
Binding matrix rows
  |

When I interrupt the merge call, I get the following readout (which may or may not be related to this problem).

In CheckDuplicateCellNames(object.list = objects) :
  Some cell names are duplicated across objects provided. Renaming to enforce unique cell names.
 Each of the 2 combined objects has sequence levels not in the other:
  - in 'x': KI270727.1
  - in 'y': GL000009.2, KI270711.1, KI270734.1
  Make sure to always combine/compare objects based on the same reference
  genome (use suppressWarnings() to suppress this warning).

Both labs used hg38.  Any thoughts?

*** update, I did 

counts$Peaks = counts$Peaks[-c(grep("GL000009.2", rownames(counts$Peaks)), grep("KI270711.1", rownames(counts$Peaks)), grep("KI270734.1", rownames(counts$Peaks))),]

and got read of the rid of the warning, but it's still hanging up.

**Comment by timoast** [comment]
Can you try the workflow shown in the [merge vignette](https://satijalab.org/signac/articles/merging.html)? (quantify the same peaks across all datasets prior to merge)
